app.py

```diff
@@ -86,6 +133,22 @@ def handleJoin(self, room, message):
             self.sendPm(user.id, self.usernotes.pendingMessages(user.id))
 
     def parseMessage(self, msg, roomName):
+        """Parses the message given by a user and delegates the tasks further
+
+        This is where we handle most of the parsing of the PS protocols. Tasks
+        like user related queries(i.e. commands) are delegated to the Command method.
+        And Showdown Tourneys are handled in their own interface in the plugins module.
```

Comment:
At least write Tournaments out fully, and also on the line above it add a space before the (